### DISCUSSION TITLE
Map keybind to multiple commands

### DISCUSSION THREAD

**Discussion by fritzrehde** [gokcehan/lf#773] [Q&A] [answered]
How can I map one keybind to multiple lf commands (not terminal commands). For example, I would like to use the following mapping: `map J updir; down; open`, expecting J to first execute updir, then down, then open. However, it only executes updir, as I am probably using the wrong syntax.

**Comment by gokcehan** [comment, marked as answer]
@fritzrehde You just need an extra `:` in the beginning as such:

```
map J :updir; down; open
```

See "Syntax" section in the documentation for more information.
